src/clr-addons/history/history-pinned.html

```diff
@@ -2,7 +2,7 @@
   <ol class="history">
     <ng-container *ngFor="let historyItem of historyElements">
       <li *ngIf="historyItem.url" class="history-item">
-        <a (click)="select(historyItem)">{{historyItem.title}}</a>
+        <a [href]="historyItem.url">{{historyItem.title}}</a>
```

Comment:
sorry forgot in last review: initially (in menu bar also) we did redirects with "location.href = ..." because IOS would lose "app mode" (when switching applications), when "installing" our apps on home screen to access it as "PWA".
so when changing this to href, we need to check this first (@d-m-s also knows this topic)

Reply:
@d-m-s requested this change - what should we do?

Reply:
1. **we need to ensure proper link styling!** 
valid "a href" seems to be required for clarity to apply styling.

1. **from ux-point of view we should enable the user to open a history-item in a new tab quickly** 
("CRTL+Click", "Left-Click > open in new Tab", ...)

1. **App-Mode must be supported and UX must not be brocken on mobile devices.**

Reply:
discussed with @derkoe, we'll go with the current commit and need to add some fixing for the app-mode in #860. 
Probably there is some possibility to prevent the app-mode from quitting when referring to another tld-url.